cves/kernel/CVE-2012-6657.yml

```diff
@@ -387,37 +382,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
+    applies: false
```

Comment:
The socket type being sent by the user/process counts as input. This input is not being checked fully. I would propose this to count under `distrust_input`.